ui/src/views/Task/ListView.vue

```diff
@@ -78,20 +78,20 @@ export default {
         })
       }
     },
-    confirmDeleteTask(taskId) {
+    confirmCancelTask(taskId) {
       Object.assign(this.dialog, {
         open: true,
-        text: `Delete task ${taskId}?`,
-        action: () => this.deleteTask(taskId)
+        text: `Stop task ${taskId}?`,
```

Comment:
```suggestion
        text: `Cancel task ${taskId}?`,
```

Reply:
Yes, that was my first attempt, but the dialog says:

```
Cancel task xxx?
[cancel] [confirm]
```

The cancel label seems inappropriate for that case